docs/modules/pkl-doc/pages/index.adoc

```diff
@@ -232,10 +232,22 @@ Relative URIs are resolved against the working directory.
 [%collapsible]
 ====
 Default: (none) +
-Example: `pkldoc`
+Example: `pkldoc` +
 The directory where generated documentation is placed.
 ====
 
+.--current-directory-mode
```

Comment:
Unless you foresee a third mode (I can’t think of one), a flag would be easier to use and understand.

Reply:
Yes, I think that it might be possible to eventually have a mode to disable `current` generation at all.

Also, I'm not very sure about a binary flag to be easier to understand to be honest. It is not disabling or enabling something here, it is changing the logic of operation between two states. Enums are supposed to be the tool for this ([one of discussions](https://stackoverflow.com/questions/4337942/using-enum-vs-boolean)).

Reply:
I agree with @translatenix here; I think this should be called `--no-symlinks`, and default to false. It's not necessarily about how the "current" dir is treated; the intention is to disable symlinks.

Reply:
That's a good point that I had failed to notice.